src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -65,6 +65,16 @@ As a result of these changes, users who are assigned will be sent an invitation
 
 You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
 
+<Alert level="note">
+
+If you use Okta to assign organization membership, you’ll be unable to make membership changes through Sentry and will need to continue using Okta.
```

Comment:
For users that are provisioned to Sentry through Okta, you'll be unable to make membership changes to them through Sentry. To remove these users, you will have to deprovision them in Okta and sync it to Sentry.